ssdp.go

```diff
@@ -534,14 +534,14 @@ func (s *Ssdp) inMSearch(st string, req *http.Request, sendTo string) {
 	if mxStr := req.Header.Get("MX"); mxStr != "" {
 		mxInt64, err := strconv.ParseInt(mxStr, 10, 0)
 		if err != nil {
-			s.logger.Warnf("Could not parse mxStr: %s", err)
-			return
-		}
-		mx = time.Duration(mxInt64) * time.Second
-		if mx < time.Second {
-			mx = time.Second
-		} else if mx > 60*time.Second {
-			mx = 60 * time.Second
+			s.logger.Warnf("Could not parse MX header: %s", err)
```

Comment:
Can you make the mxInt64 default match our minimum (or other) time?

Reply:
That is... when we fail to parse their value.

Reply:
Fixed